7.3.частые_сценарии/7.3.10.py

```diff
@@ -0,0 +1,6 @@
+total = 1                    
+for i in range(10):          
+    num = int(input())       
+    if num > 0:              
+        total = num * total  
```

Comment:
Не забывай использовать расширенные операторы присваивания:
```suggestion
        total *= num
```